ambry-server/src/main/java/com/github/ambry/server/AmbryRequests.java

```diff
@@ -494,6 +496,74 @@ public void handleDeleteRequest(NetworkRequest request) throws IOException, Inte
             metrics.deleteBlobTotalTimeInMs, null, null, totalTimeSpent));
   }
 
+  @Override
+  public void handlePurgeRequest(NetworkRequest request) throws IOException, InterruptedException {
+    PurgeRequest purgeRequest = PurgeRequest.readFrom(new DataInputStream(request.getInputStream()), clusterMap);
```

Comment:
Is there any possibility, the purge request will be handled locally? Shall we keep the code consistent with other request handler to differentiate localrequest vs networkrequest?